src/orca/services/base/ops.py

```diff
@@ -1,27 +1,38 @@
 from abc import ABC, abstractmethod
-from typing import Any, Generic, TypeVar
+from typing import Any, ClassVar, Generic, Type, TypeVar
 
 from pydantic import ConfigDict
 from pydantic.dataclasses import dataclass
 from typing_extensions import Self
 
-from orca.services.base.config import BaseServiceConfig
+from orca.services.base.config import BaseConfig
 
 ClientClass = TypeVar("ClientClass", bound=Any)
 
-ServiceConfig = TypeVar("ServiceConfig", bound=BaseServiceConfig)
+ConfigClass = TypeVar("ConfigClass", bound=BaseConfig)
 
 
 @dataclass(kw_only=False, config=ConfigDict(arbitrary_types_allowed=True))
-class BaseOps(ABC, Generic[ServiceConfig, ClientClass]):
-    """Base collection of operations for a service."""
+class BaseOps(ABC, Generic[ConfigClass, ClientClass]):
+    """Base collection of operations for a service.
+
+    N.B. Make sure to override the type hint for the ``client`` class
```

Comment:
TIL: N.B means: nota bene.  It is used to emphasize an important point.

Reply:
I didn't realize that was the spelt out version. I read it in Italian in my mind, which sounds nice. I'm going to read it that way going forward (instead of just thinking of the letters N and B).